proposals/module-linking/Binary.md

```diff
@@ -69,12 +69,12 @@ Updates to
 [`importdesc`](https://webassembly.github.io/spec/core/binary/modules.html#binary-importdesc)
 
 ```
-# note that in MVP wasm this encoding specifies a zero-length field name, but
+# note that in MVP wasm this encoding specifies a zero-length `module` name, but
```

Comment:
I think this `module` refers to [`name`](https://webassembly.github.io/spec/core/binary/modules.html#import-section), right?

Reply:
Oops, right.